test/allowlist_for_publicAPI.json

```diff
@@ -231,6 +232,9 @@
     "urlunparse"
   ],
   "torch.distributed.rpc": [],
+  "torch.distributed.tensor": [
+    "DeviceMesh"
```

Comment:
New entries here are not allowed either. Please make the corresponding class properly public.

Reply:
done :)